targets/ESP32/_nanoCLR/System.IO.Ports/sys_io_ser_native_System_IO_Ports_SerialPort.cpp

```
@@ -1,1467 +1,1467 @@
+
+    NANOCLR_NOCLEANUP();
+}
+
+HRESULT Library_sys_io_ser_native_System_IO_Ports_SerialPort::GetDeviceSelector___STATIC__STRING(
+    CLR_RT_StackFrame &stack)
+{
+    NANOCLR_HEADER();
+
+    // declare the device selector string whose max size is "COM1,COM2,COM3" + terminator
+    // and init with the terminator
+    static char deviceSelectorString[] =
```

cppcheck

[error] Syntax Error

(internalAstError)